### PR TITLE
Add answers to questions from Luxor to manual/clars

### PR DESCRIPTION
**Pull request by vmcj**
I'll squash after approval.

No reason to merge it now, but I found it in the GDrive folder.

### CODE REVIEW

File: etc/db-config.yaml [hunk 1]
```diff
@@ -186,6 +186,12 @@
             default_value:
                 - No comment.
                 - Read the problem statement carefully.
+                - The hardware of the machines evaluating your submissions is identical to the team machines.
+                  They run the same operating system and base image but are tuned to make submission run times
+                  more consistent, for example we pin your submission to one core.
+                - DOMjudge does not define any special macros like ONLINE_JUDGE.
+                - For interactive problems, only your code runs inside of the sandbox and other processes
+                  including the interactive problem interactor do not count towards memory or time limits for the submission.
```
Comment (eldering) [inline etc/db-config.yaml:194]: I don't really understand what these do here in the default clarification answers (I think they should have been general clarifications sent out at the beginning of WFs), but they should not go into `main`.

File: doc/manual/team.rst [hunk 2]
```diff
@@ -261,6 +261,8 @@ your program).
 On the contest control system your program enters a queue, awaiting compilation,
 execution and testing on one of the autojudges.
 
+Beware that if you submit multiple times in a short time interval the DOMjudge will determine which submission to judge based on both the submission time and how many other submission you have in queue to penalize spamming.
```
Comment (eldering) [inline doc/manual/team.rst:264]:
```suggestion
Beware that if you submit multiple times in a short time interval, then DOMjudge will determine which submission to judge based on both the submission time and how many other submission you have in queue to penalize spamming.
```

Comment (eldering) [reply doc/manual/team.rst:264]: Also, I'm not really sure this should go into the manual in `main`.

File: doc/manual/team.rst [hunk 3]
```diff
@@ -410,6 +418,8 @@ for different programming languages.
 Improvements to DOMjudge
 ------------------------
 
-The DOMjudge team would like your feedback. We do not receive much feedback from participants.
-If you find something lacking or have improvement ideas, please report them. See https://www.domjudge.org/development.
+The DOMjudge team would like your feedback. We do not
+receive much feedback from participants. If you find
+something lacking or have improvement ideas, please report
+them. See https://www.domjudge.org/development.
```
[the remark this answers is not in the document]
Comment (vmcj) [reply doc/manual/team.rst:424]: Only reformatting.

**Pull request closed without merging**